packages/playground/windows/playground-composition/Playground-Composition.cpp

```diff
@@ -480,6 +480,9 @@ LRESULT CALLBACK WndProc(HWND hwnd, UINT message, WPARAM wparam, LPARAM lparam)
     case WM_WINDOWPOSCHANGED: {
       auto windowData = WindowData::GetFromWindow(hwnd);
       windowData->UpdateSize(hwnd);
+
+      winrt::Microsoft::ReactNative::ReactNotificationService rns(windowData->InstanceSettings().Notifications());
+      winrt::Microsoft::ReactNative::ForwardWindowMessage(rns, hwnd, message, wparam, lparam);
```

Comment:
@jonthysell  we might be switching away from this API in the near future, but in the meantime, we might need to add this to the template.

Reply:
Do we have an issue to switch this over to a real API that takes a size.  Something like MS::RN::NotifyOfWindowSizeChanged(rns, size);

Reply:
Opened #12497 to track the work